test/ShapeCrawler.Tests.Unit/ShapeFillTests.cs

```diff
@@ -171,6 +171,30 @@ public void SetColor_sets_solid_color(string file, int slideNumber, string shape
         pres.Validate();
     }
 
+
+    [Test]
+    [TestCase("autoshape-case005_text-frame.pptx", 1, "AutoShape 1")]
+    [TestCase("autoshape-case005_text-frame.pptx", 1, "AutoShape 2")]
+    [TestCase("autoshape-grouping.pptx", 1, "AutoShape 1")]
+    public void SetColor_sets_nofill(
+        string file,
+        int slideNumber,
+        string shapeName
+    )
+    {
+        // Arrange
+        var pres = new Presentation(StreamOf(file));
+        var shape = pres.Slides[slideNumber - 1].Shapes.GetByName(shapeName);
+        var shapeFill = shape.Fill;
+
+        // Act
+        shapeFill.SetNoFill();
+
+        // Assert
+        Type_returns_fill_type(shape, FillType.NoFill);
```

Comment:
@brunnobb, thank you for the PR.

I believe that calling a unit test inside another unit test is not good idea. For example, it leads to Loss Of Isolation: Cascading failures if one test breaks, affecting others that depend on it.

Could you please change such assertions. I think like `shape.Fill.Type.Should().Be(FillType.NoFill);` should be enough.

Reply:
Seems good.

I still have a few changes this week, and that will require another PR.
I`ll go back and work on this after the end of our sprint here and submit the changes, as we are currently running on a fork

Reply:
New commit with this change is done